Makefile

```diff
@@ -100,10 +88,14 @@ else
 KUSTOMIZE=$(shell which kustomize)
 endif
 
-generate-metallb-manifests:
+generate-metallb-manifests:  ## Generate metallb manifests
 	@echo "Generating MetalLB manifests"
 	hack/generate-metallb-manifests.sh
 
-validate-metallb-manifests:
+validate-metallb-manifests:  ## Validate metallb manifests
 	@echo "Comparing newly generated MetalLB manifests to existing ones"
 	hack/compare-gen-manifests.sh
+
+help:  ## Show this help
```

Comment:
Can you also add a line documenting the default behaviour (which is make manager)?

Reply:
are u referring to all target ? I am not sure what is the ask ?

Reply:
Yes exactly, I am talking about what happens if you just run `make`